cdc/entry/mounter.go

```diff
@@ -490,36 +489,36 @@ func formatColVal(datum types.Datum, tp byte) (
 		const sizeOfV = unsafe.Sizeof(v)
 		return v, int(sizeOfV), warn, nil
 	default:
+		// NOTICE: GetValue() may return some types that go sql not support, which will cause sink DML fail
+		// Make specified convert upper if you need
+		// Go sql support type ref to: https://github.com/golang/go/blob/go1.17.4/src/database/sql/driver/types.go#L236
 		return datum.GetValue(), sizeOfDatum(datum), "", nil
 	}
 }
 
-func getDefaultOrZeroValue(col *timodel.ColumnInfo) (interface{}, int) {
-	// see https://github.com/pingcap/tidb/issues/9304
-	// must use null if TiDB not write the column value when default value is null
-	// and the value is null
+func getDefaultOrZeroValue(col *timodel.ColumnInfo) (interface{}, int, string, error) {
```

Comment:
Could you add some comments to list all possible results behind the `interface{}`? Also add to `formatColVal` and `model.Column.Value`.

Reply:
Get. OK.